src/DataProtection/DataProtection/src/TypeForwardingActivator.cs

```diff
@@ -13,7 +12,6 @@ internal class TypeForwardingActivator : SimpleActivator
         private const string OldNamespace = "Microsoft.AspNet.DataProtection";
         private const string CurrentNamespace = "Microsoft.AspNetCore.DataProtection";
         private readonly ILogger _logger;
-        private static readonly Regex _versionPattern = new Regex(@",\s?Version=[0-9]+(\.[0-9]+){0,3}", RegexOptions.Compiled, TimeSpan.FromSeconds(2));
```

Comment:
This change is from https://github.com/dotnet/aspnetcore/issues/30013 which is unrelated, but we are rolling into this PR